temporalio/ui
# Individual Event History page

## Description & motivation 💭 
This PR creates an individual workflow event page, allowing a user to share specific events via URL to see the details of that event quickly without needing to scroll through event history. If the event id in the URL does not exist in the event history, the page will show a 'No Events Match' result.

This also changes the event summary row opening behavior to allow multiple open rows. It removes auto-opening rows with keyboard navigation.

### Screenshots (if applicable) 📸 

<img width="1728" alt="Screen Shot 2023-10-17 at 10 36 50 AM" src="https://github.com/temporalio/ui/assets/7967403/8f671cf0-3db8-421e-ab93-f22420bd55df">
<img width="1728" alt="Screen Shot 2023-10-17 at 2 55 33 PM" src="https://github.com/temporalio/ui/assets/7967403/f64ddbc6-3a95-43ca-9295-3fb87a186629">
<img width="1727" alt="Screen Shot 2023-10-17 at 2 57 22 PM" src="https://github.com/temporalio/ui/assets/7967403/4fad32cd-3a4a-4a6e-a702-109d0c5adc5e">

### Design Considerations 🎨 
Would love to pair with @tigernaut and see their opinions on user flow. 

## Testing 🧪 

### How was this tested 👻 


### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
- go to a workflow, select the id number for the workflow, this should take you to that specific page for the selected workflow event. If copy and pasted this should take you to that same event page. 

## Checklists

### Draft Checklist 

### Merge Checklist 

### Issue(s) closed 
https://temporalio.atlassian.net/browse/DT-1517
## Docs

### Any docs updates needed?
